src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__img-wrapp">
+        <img
+          class="product-card__img"
+          alt="imac photo"
+          src="images/imac.jpeg">
+      </div>
+      <header class="product-card__header">
```

Comment:
[Code style](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting), Add empty lines between multiline sibling blocks of HTML
Check all over the document